### DISCUSSION TITLE
Использование ресурса OrganizationAffilation

### DISCUSSION THREAD

**Discussion by eugenekogan-spb** [fhir-ru/core#60] [Административная РГ]
Профиль Взаимосвязь организаций
Используется для указания неадминистративной взаимосвязи между юридическими лицами либо подразделениями
Например, в следующих случаях:
- юридические лица входят в состав (зависимы от) другого юридического лица, - когда эта информация полезна для аналитики, группировки данных в отчетах
- некоторые подразделения одной или нескольких организаций входят в функциональное объединение (например, центр лечения боли на базе нескольких отделений больницы и кафедры ВУЗа)

Возможный атрибут - Тип взаимосвязи

Дополнительно:

в структуру OrganizationAffilation входит указание HealthcareService, для указания категории медицинских услуг, которые организация выполняет в данной роли.

**Comment by Avazkh** [comment]
Как указано в FHIR: "A relationship between 2 organizations over a period of time, where the entities are separate business entities..."

**Reply by Avazkh** [reply]
Не будет ли это существенным ограничием для использования? Только для страхования есть атрибут network. Или я не верно понимаю?

**Comment by shamilnn** [comment]
Добавлял картинку структуры, с _OrganizationAffiliation_ в [Маппинг данных ФРМО на FHIR ресурсы](https://github.com/fhir-ru/core/discussions/26):

![image](https://user-images.githubusercontent.com/75683852/146922323-6de6ec27-7e4c-49d8-8028-798cc0414115.png)
